ios/ReactTestApp/React+Compatibility.m

```diff
@@ -25,3 +48,37 @@ void RTATriggerReloadCommand(RCTBridge *bridge, NSString *reason)
     RCTTriggerReloadCommandListeners(reason);
 #endif
 }
+
+// MARK: - [0.63.2] Images do not render on iOS 14
+// See https://github.com/facebook/react-native/pull/29420
+
+#if !TARGET_OS_OSX && REACT_NATIVE_VERSION < 6302
+
+#import <React/RCTUIImageViewAnimated.h>
+
+@implementation RCTUIImageViewAnimated (ReactTestApp)
+
++ (void)initialize
+{
+    if ([self class] != [RCTUIImageViewAnimated class]) {
+        return;
+    }
+
+    if (@available(iOS 14.0, *)) {
+        RTASwizzleSelector([self class], @selector(displayLayer:), @selector(rta_displayLayer:));
+    }
+}
+
+- (void)rta_displayLayer:(CALayer *)layer
+{
+    if ([self respondsToSelector:@selector(currentFrame)] &&
+        [self performSelector:@selector(currentFrame)] == nil) {
+        [super displayLayer:layer];
```

Comment:
Took me a minute to figure this out :).   Maybe we should have a comment containing the source of the original displayLayer: method.

The 62 version is
```objc
  if (_currentFrame) {
    layer.contentsScale = self.animatedImageScale;
    layer.contents = (__bridge id)_currentFrame.CGImage;
  }
```
and the post 62 version adds
```objc
else {
    [super displayLayer:layer];
  }
```
so the logic here is doing the opposite of the original `if` and calling the post-62 `else` clause, otherwise call `[self rta_displayLayer:layer]` which I think is intended to call the original RCTUIImageViewAnimated displayLayer.   That's the case for a regular category, but for a swizzle, its not correct because something else *could* have also swizzled the method.   You have to store the `originalMethod` from `RTASwizzleSelector` and call it.   We had a bug in Office where RCT was swizzling a method for the shake gesture, but Office was also swizzling the method, so the RCT swizzle made the Office method no longer callable.   So in the react-native-macos fork I changed the RCTSwizzle method to return the previous selector and store it.   The RTASizzle should do the same so we preserve any chain of swizzles.